app/controllers/journal_entries_controller.rb

```diff
@@ -0,0 +1,50 @@
+class JournalEntriesController < ApplicationController
+
+  def index
+    @journal_entries = JournalEntry.all
+    redirect_to journals_path
+  end
+  
+  def new
+    @journal = Journal.find(params[:journal_id])
+    @journal_entry = @journal.journal_entries.new
+  end
+
+  def create
+    @journal = Journal.find(params[:journal_id])
+    @journal_entry = @journal.journal_entries.new(journal_entry_params)
+
+    if @journal_entry.save
+      redirect_to journal_path(@journal), notice: 'Journal entry was successfully created.'
+    else
+      render :new
+    end
+  end
+
+  def destroy
+    @journal = Journal.find(params[:journal_id])
+    @journal_entry = @journal.journal_entries.find(params[:id])
+    @journal_entry.destroy
```

Comment:
What if destroy operation fails? Please handle that case as well. (Hint: study about `destroy!`)